src/windows/CameraProxy.js

```diff
@@ -310,29 +319,50 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
 
         tempFolder.createFileAsync(fileName, generateUniqueCollisionOption)
             .then(function(tempCapturedFile) {
-                capturedFile = tempCapturedFile;
-                return capture.capturePhotoToStorageFileAsync(encodingProperties, capturedFile);
+                return new WinJS.Promise(function (complete) {
+                    var imgStream = new Windows.Storage.Streams.InMemoryRandomAccessStream();
```

Comment:
imgStream.close() needs to be invoked once you are done using it to release system resources. It needs to be part of the success and error case.